docs/new/posts/PostgreSQL/postgresql-q3-2025-release.md

```diff
@@ -0,0 +1,20 @@
+---
+date: 2025-09-18
+description: >
+  Percona Distribution for PostgreSQL 16.10, 15.14, 14.19, 13.22 have been released.
+authors: [Andriciuc]
+categories:
+  - PostgreSQL
+tags:
+  - Percona Distribution for PostgreSQL
+---
+
+# Percona Distribution for PostgreSQL 16.10, 15.14, 14.19, 13.22 have been released
+
+<!-- more -->
+
+The releases of [Percona Distribution for PostgreSQL](https://docs.percona.com/postgresql/16/index.html){:target="_blank"} 16.10, 15.14, 14.19 and 13.22 have been released throughout September 2025. They follow the release of Percona Distribution for PostgreSQL 17.6 and complete the PostgreSQL minor version release cycle for Q3 2025.
```

Comment:
Could this be an unordered list instead of a comma list? It may be easier for users to read.

The following releases of [Percona Distribution for PostgreSQL](https://docs.percona.com/postgresql/16/index.html){:target="_blank"} were published in September 2025:

- 16.10  
- 15.14  
- 14.19  
- 13.22  

These releases follow version 17.6 and complete the PostgreSQL minor version release cycle for Q3 2025.